content/python/concepts/functions/terms/arguments-parameters/arguments-parameters.md

```diff
@@ -0,0 +1,154 @@
+---
+Title: 'Arguments/Parameters'
+Description: 'Supplies data to a defined function when it is called in a program.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Functions'
+  - 'Methods'
+  - 'Parameters'
+  - 'Arguments'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+
+If parameters are defined inside a function, then any data passed into the function later in a program are known as arguments.
+
+## Parameters
+
+Parameters are variables that are declared in the function definition. They are usually processed in the function body to produce the desired result. When the function is called, each parameter is assigned the value which was passed as a corresponding argument.
+
+For example, the function below contains parameters for a `character`, a `setting`, and a `skill`, which are used as inputs to write the first sentence of a book.
+
+```py
+def write_a_book(character, setting, skill):
+  print(character + " is in " +
+        setting + " practicing " +
+        skill + ".")
+
+write_a_book("Naomi", "engineering school", "welding")
+```
+
+The output will look like this:
+
+```shell
+Naomi is in engineering school practicing welding.
+```
+
+## Default Parameter Values
+
+Function parameters can also be initialized to a default value. In the `calc_total()` function, there are `amount` and `discount` parameters.
+
+- When the `discount` value is explicitly specified in the function call, that value is used.
+- Otherwise, the default value of 10 is used.
+
+```codebyte/python
+def calc_total(amount, discount=10):
+  total = amount * (1 - 0.01 * discount)
+  return total
+
+calc_total(100)     # Output: 90.0
```

Comment:
Should lines 52 and 53 be placed in a print statement to actually print out the totals when it's executed?

Reply:
Absolutely! Thanks. 😄